discovery-provider/ddl/migrations/0010_usdc_userbank_tables.sql

```diff
@@ -0,0 +1,62 @@
+BEGIN;
+
+-- USDC user bank accounts
+CREATE TABLE IF NOT EXISTS usdc_user_bank_accounts (
+    "signature" varchar NOT NULL,
+    ethereum_address varchar NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    bank_account varchar NOT NULL,
+    PRIMARY KEY("signature")
+);
+CREATE INDEX IF NOT EXISTS idx_usdc_user_bank_accounts_eth_address ON usdc_user_bank_accounts USING btree (ethereum_address);
+
+-- USDC transactions history
+CREATE TABLE IF NOT EXISTS usdc_transactions_history (
+    user_bank varchar NOT NULL,
+    slot integer NOT NULL,
+    "signature" varchar NOT NULL,
+    transaction_type varchar NOT NULL,
```

Comment:
this is varchar in existing `audio_transaction_history` table, but design doc has enum. What do we think?

Reply:
Oh interesting, the [model class for autio transactions](https://github.com/AudiusProject/audius-protocol/blob/main/discovery-provider/src/models/users/audio_transactions_history.py#L29) specifies an enum. So I wonder how it ended up as a varchar in the table definition...

Some basic research tells me there are special semantics with enums that we may not want:
* default ordering is the order the enum values are specified in during table creation
* You can't compare an enum type against a different enum type, even if the string values match. I _think_ this isn't a problem for comparing a string to an enum value (i.e. `where transaction_type = 'PURCHASE'`)

And enums occupy less space on disk, so there's that.
Probably better to keep it consistent with the other table though.